### ISSUE TITLE
add a guide to the documentation page on rtd

### ISSUE THREAD

**Issue by kendonB** [dictation-toolbox/Caster#572]
Currently, there doesn't seem to be a page explaining the syntax of the commands. For example, a new user reading this:

```
(go | move) <direction> [<number_of_lines_to_search>] [<before_after>] [<occurrence_number>] <dictation>
```
would have no idea what the `(|<[` etc. means. We should explain this on a documentation page and provide links to this page throughout the documentation we are relevant.

**Comment by LexiconCode** [comment]
I agree we could create a quick cheat sheet for dragonfly grammars similar to how it's already set up. Explain dragonfly first then caster additions.
